root/usr/bin/enable-lldp

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+enable_lldp_int() {
+  local int=$1
+  shift
+  local ip=$1
+  shift
+
+  variables="$@"
+
+  lldptool -L -i $int adminstatus=rxtx
+
+  for var in $variables; do
+    lldptool -i $int -T -V $var enableTx=yes
+  done
+
+}
+
+for inter in $(ls /sys/class/net/ | grep -v "lo"); do
+ enable_lldp_int $inter $ipaddress portDesc sysName sysDesc sysCap
+done
```

Comment:
This command is executed for each individual device as a parameter. Remove the loop here please, you will have the device name as first parameter.

```shell
    dev=$(basename $1)
    [[ "$dev" == "lo" ]] && return
    enable_lldp_int $dev ...
```
Whole script can be simplified a lot. Also please add output (it will go to journald):

```
echo "Configuring LLDP on interface $dev"
```

So we can check the logs if it gets called properly.